skbio/diversity/beta/_base.py

```diff
@@ -8,28 +8,31 @@
 
 from __future__ import absolute_import, division, print_function
 
+from functools import partial
+
 import numpy as np
 from scipy.spatial.distance import pdist, squareform
 
+from skbio.diversity.beta._unifrac import unweighted_unifrac, weighted_unifrac
 from skbio.stats.distance import DistanceMatrix
 from skbio.util._decorator import experimental, deprecated
 
 
 @experimental(as_of="0.4.0")
-def pw_distances(counts, ids=None, metric="braycurtis"):
+def pw_distances(metric, counts, ids=None, **kwargs):
     """Compute distances between all pairs of columns in a counts matrix
 
     Parameters
     ----------
+    metric : str, callable
+        The pairwise distance function as a string or callable to use when
+        generating pairwise distances. See the scipy ``pdist`` docs and the
+        scikit-bio functions linked under *See Also* for available metrics.
     counts : 2D array_like of ints or floats
```

Comment:
just a note, and likely out of scope for the PR, but it would be awesome if sparse vectors were allowed here